pyproject.toml

```diff
@@ -7,6 +7,7 @@ name = "ansible-wisdom"
 description = "Ansible Lightspeed with IBM watsonx Code Assistant."
 version = "0.1.0"
 dependencies = [
+  'aiohttp~=3.9.4',
```

Comment:
`aiohttp` is not a direct dependency of our service. It should not be in this list.

Reply:
Updated in the latest commit. Thanks.